### DISCUSSION TITLE
How to bind Control Tab to cycle through tabs?

### DISCUSSION THREAD

**Discussion by Dialga** [zellij-org/zellij#3317] [Q&A]
How to bind Control Tab to cycle through tabs? Also bind Ctrl+Shift+Tab to cycle reverse order?

**Comment by Lockszmith-GH** [comment]
Reminded me of a previous discussion - #2478 where [somebody had a similar sentiment](https://github.com/zellij-org/zellij/issues/2478#issuecomment-1872593031).
